content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -40,6 +40,9 @@ The cluster recovery process includes:
 - We recommend that you use clean persistent storage drives for the new cluster.
     If you use the original storage drives,
     make sure that you backup the files on the original storage drives to a safe location.
+- The cluster configuration file that you want to use as the configuration for the cluster.
+    The cluster configuration file is `/css/ccs-redis.rdb` on each of the nodes.
```

Comment:
This paragraph is not accurate (i.e. this sentence and the next one), as the file is located on the persistent storage drive for each node. It's not necessarily on the node itself.

Reply:
Change to; The cluster configuration file is `/css/ccs-redis.rdb` on the persistent storage drive for each node. OK?